src/js/files/components/UploadItem.js

```diff
@@ -26,16 +27,51 @@ const UploadItemTitle = styled.div`
     }
     span:last-child {
         margin-left: auto;
+        color: ${props => (props.failed ? getColor({ theme: props.theme, color: "red" }) : "inherit")};
+        font-size: ${props => (props.failed ? getFontSize("sm") : "inherit")};
+        font-style: ${props => (props.failed ? "italic" : "normal")};
```

Comment:
Italics are not used anywhere else in Virtool (AFAIK) and I don't consider it part of the design language.

For this, why don't we do the following to update the styling:

1. Use thick font-weight for the file name.
2. Use regular font-weight, style, and size for **Failed**.
3. Use `lg` font-size and thick weight for the upload overlay title (**Uploads (4)**).

Reply:
Changed to use your suggested styling.

Updated overlay: 
![image](https://user-images.githubusercontent.com/59776400/154318927-6e795b37-7215-4a06-bdc9-ca7a2734072a.png)

Reply:
Thanks for the screenshot. Looks great and integrated with the rest of the UI.